jbmc/regression/jbmc-strings/ConstantEvaluationReplace/Main.java

```diff
@@ -0,0 +1,90 @@
+public class Main {
+    public void replace() {
+        String str = "abded";
+        str = str.replace('d', 'c');
+        assert str.equals("abcec");
+    }
+
+    public void notEqualsReplace() {
+        String str = "abded";
+        str = str.replace('d', 'c');
+        assert !str.equals("abdec");
+    }
+
+    public void noReplace() {
+        String str = "abcde";
+        str = str.replace('z', 'c');
+        assert str.equals("abcde");
+    }
+
+    public void randomCharReplace(char b) {
+        String str = "abcde";
+        str = str.replace('a', b);
+        assert str.equals("abcde");
+    }
+
+    public void randomCharsReplace(char a, char b) {
+        String str = "abcde";
+        str = str.replace(a, b);
+        assert str.equals("abcde");
+    }
+
+    public void nopropReplace(String str) {
+        str = str.replace('d', 'c');
+        assert str.equals("abcec");
+    }
+
+    public void stringReplace() {
+        String str = "abded";
```

Comment:
Duplicate of `replace`

Reply:
One takes 2 characters, other takes 2 strings.